src/core/spmatrix.c

```diff
@@ -670,7 +674,7 @@ static int igraph_i_spmatrix_cleanup(igraph_spmatrix_t *m) {
 
 /**
  * \function igraph_spmatrix_clear_col
- * \brief Clears a column in the matrix (sets all of its elements to zero)
+ * \brief Clears a column in the matrix (sets all of its elements to zero).
  * \param m The matrix.
  * \param col The index of the column to be cleared.
  * \return Error code. The current implementation always succeeds.
```

Comment:
This is not true any more, the current implementation can return `IGRAPH_EINVAL` if the column index is invalid.